Assets/Scripts/Gameplay/Action/Input/ActionLogic.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Actions
+{
+    /// <summary>
+    /// List of all Types of Actions. There is a many-to-one mapping of Actions to ActionLogics.
+    /// </summary>
+    public enum ActionLogic
```

Comment:
I thought we didn't need this anymore with your action ID setup?

Reply:
Oh right, this is the action logic, not action type. Could the same apply here? Do we still need this enum?

Reply:
We do need it - it's used to cancel things based on their common logic (and it could be the case that several different action types share the logic type, as it is for Emotes and some melee attacks).